LiskHQ/lisk-desktop
# Add POM transaction information - Closes #3686

### What was the problem?
This PR resolves #3686

### How was it solved?
The transactions page crashes because
 - POM tx information is missing.
 - The Account visual component is not prepared for missing tx types.
I fixed both.

### How was it tested?
Manually.
- Check The transactions page ([devURL](http://localhost:8080/#/transactions)) for POM txs.
- Check POM tx details page. ([This](http://localhost:8080/#/transactions?modal=transactionDetails&transactionId=a351e695bac141daf94da1307156969a763df3566918d04c4639ba8f772e2988&token=LSK) on testnet)
